### PR TITLE
update instances of `/r` for `@Damage` for en.json

### PR DESCRIPTION
**Pull request by ammalagonc**
There was only one instance in this file. Will do a separate pull request for the `re-en.json` update as that is the main offender.

### CODE REVIEW

File: static/lang/en.json [hunk 1]
```diff
@@ -734,7 +734,6 @@
         "ArmorPropertyRuneEthereal": "Ethereal",
         "ArmorPropertyRuneFireResistant": "Fire-Resistant",
         "ArmorPropertyRuneFortification": "Fortification",
-        "ArmorPropertyRuneGlamered": "Glamered",
```
Comment (Cuingamehtar) [inline static/lang/en.json:737]: Is there a reason that Glamered rune is gone?

Comment (Cuingamehtar) [reply static/lang/en.json:737]: I see, it's from an unrelated PR. You should probably remove this change from here and keep it in that PR.

Comment (ammalagonc) [reply static/lang/en.json]: Removed the change.

File: static/lang/en.json [hunk 2]
```diff
@@ -2807,7 +2807,7 @@
...
+                    "Buck": "<hr />\n<p>Most monsters that serve as mounts can attempt to buck off unwanted or annoying riders, but most mounts will not use this reaction against a trusted creature unless the mounts are spooked or mistreated.</p>\n<p><strong>Trigger</strong> A creature @UUID[Compendium.pf2e.actionspf2e.Item.PM5jvValFkbFH3TV]{Mounts} or uses the @UUID[Compendium.pf2e.actionspf2e.Item.q9nbyIF0PEBqMtYe]{Command an Animal} action while riding the monster.</p>\n<hr />\n<p><strong>Effect</strong> The triggering creature must succeed at a Reflex saving throw against the listed DC or fall off the creature and land @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}. If the save is a critical failure, the triggering creature also takes @Damage[1d6[bludgeoning]]{1d6 bludgeoning damage} in addition to the normal damage for the fall.</p>",
```
Comment (TikaelSol) [inline static/lang/en.json:2810]:
```suggestion
                    "Buck": "<hr />\n<p>Most monsters that serve as mounts can attempt to buck off unwanted or annoying riders, but most mounts will not use this reaction against a trusted creature unless the mounts are spooked or mistreated.</p>\n<p><strong>Trigger</strong> A creature @UUID[Compendium.pf2e.actionspf2e.Item.PM5jvValFkbFH3TV]{Mounts} or uses the @UUID[Compendium.pf2e.actionspf2e.Item.q9nbyIF0PEBqMtYe]{Command an Animal} action while riding the monster.</p>\n<hr />\n<p><strong>Effect</strong> The triggering creature must succeed at a Reflex saving throw against the listed DC or fall off the creature and land @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}. If the save is a critical failure, the triggering creature also takes @Damage[1d6[bludgeoning]] damage in addition to the normal damage for the fall.</p>",
```

**Pull request merged**